doc/howto/usage/cluster/k8s-aws/README.md

```diff
@@ -0,0 +1,650 @@
+#PaddlePaddle on AWS with Kubernetes
+
+##Prerequisites
+
+First, you need an AWS account, please check out [this](http://docs.aws.amazon.com/lambda/latest/dg/setting-up.html) for how to setup an AWS account. 
```

Comment:
your AWS account ==> our AWS account

As this is a tutorial, we are suppose to run it together with our readers.